scania-digital-design-system/sdds
# fix(foundation-stories): code formatting and examples improvements

**Describe pull-request**  
Code formatting and examples improvements in foundation stories

**How to test**  
1. Open Netlify link below
2. Browse through foundation stories and see if code examples look good (spacing, demo comments and typography text examples)
